src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrUpdater.cs

```diff
@@ -171,6 +176,8 @@ public class VmrUpdater : VmrManagerBase, IVmrUpdater
                 tpnTemplatePath, 
                 cancellationToken);
         }
+
+        CleanUpRemovedRepos(readmeTemplatePath, tpnTemplatePath);
```

Comment:
I don't think this is the right place to call this for several reasons:
1. During the recursive sync, changes like these need to be committed on the sync branch
2. I would just do this during the recursive sync
3. I don't actually see these changes are committed at the moment? How does that happen? We should ideally have a commit for each removed repo (or just for all of them, that's okay)

Reply:
might be better to have one commit for all as there are common files, but I'm not sure about it

Reply:
I think it doesn't matter from UX point of view so whatever is easier to put down into code